README.md

```diff
@@ -1,142 +1,126 @@
 # `Frontend`
 
-## Prerequisite
+## Pre-requisites
+
+- Install Java OpenJDK 11
```

Comment:
The instructions related to java is not for the frontend, but for the other apps. Should we refer to the wiki in sklintyg/common instead? Similar to what you do for the sklintyg/devops?

Reply:
I thought about it, but not all instructions for backend are necessary. I felt it would be least confusing this way, but perhaps we should structure the docs for `sklintyg/common` a little better instead?

Reply:
I get your point about it being less confusing. Keep it as you have written and we can change it when we decide to restructure sklintyg/common wiki.